example/Vagrantfile

```diff
@@ -5,100 +5,134 @@
-          end
+            v.customize ["modifyvm", :id, "--nicpromisc2", "allow-all"]
         end
 
         # Create a public network, which generally matched to bridged network.
         # Bridged networks make the machine appear as another physical device on
         # your network.
         # target.vm.network :public_network
+        if ENV['WORKSPACE']
+          target.vm.synced_folder "#{ENV['WORKSPACE']}/build-deps", "/home/vagrant/src/"
+          target.vm.synced_folder "#{ENV['WORKSPACE']}/build-config", "/home/vagrant/src/build-config"
+          if ENV['MULTI'] =="true"
+            target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/build"
+            for repo in ENV['REPO_NAME'].split(' ')
+              if repo != "rackhd"
+                target.vm.synced_folder "#{ENV['WORKSPACE']}/#{repo}", "/home/vagrant/src/#{repo}"
+              end
+            end
+            if ENV['REPO_NAME'].include? "on-tasks"
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/on-tasks", "/home/vagrant/src/on-taskgraph/node_modules/on-tasks/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/on-tasks", "/home/vagrant/src/on-http/node_modules/on-tasks/" 
+            end
+            if ENV['REPO_NAME'].include? "on-core"
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/on-core", "/home/vagrant/src/on-taskgraph/node_modules/on-core/"  
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/on-core", "/home/vagrant/src/on-http/node_modules/on-core/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/on-core", "/home/vagrant/src/on-tftp/node_modules/on-core/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/on-core", "/home/vagrant/src/on-dhcp-proxy/node_modules/on-core/" 
+            end
+          else
+            if ENV['REPO_NAME']
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/#{ENV['REPO_NAME']}"
+            end
+            if ENV['REPO_NAME'] == "on-tasks"
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-taskgraph/node_modules/on-tasks/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-http/node_modules/on-tasks/" 
+            end
+            if ENV['REPO_NAME'] == "on-core"
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-taskgraph/node_modules/on-core/"  
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-http/node_modules/on-core/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-tftp/node_modules/on-core/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-dhcp-proxy/node_modules/on-core/" 
+            end
+          end
+        end
+        
+        if ENV['CONFIG_DIR']
+          target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['CONFIG_DIR']}", "/opt/monorail/"
```

Comment:
This line of code can be kept.
A RackHD config.json file for users can be put in the same directory for replacing default  rackhd config.